src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2799,6 +2815,31 @@ public Problem loadCCSProblemFiles(IInternalContest contest, String dataFileBase
         return problem;
     }
 
+    protected void loadDataFiles(Problem problem, ArrayList<SerializedFile> dataFiles, ArrayList<SerializedFile> answerFiles, String sampleDataDirectory, boolean loadExternalFile) {
```

Comment:
Since this method is being used to load ONLY sample data files, for clarity I would suggest a better method name would be `loadSampleDataFiles`.  Although actually I agree with JohnB's earlier comment that the most logical thing to do would be to use the same method for loading both samples and secrets.